Added pareto principle achievement

Ciock opened the pull request:
Inspired by `Variety is the Spice of Life` achievement, I thought about a `Pareto principle` achievement.

Basically it's a check that the top 20% artists contribute to at least 80% streamed hours played

closes #10

willyw0nka committed:
Fixed builder

willyw0nka committed:
Fixed builder

willyw0nka committed:
Merge pull request #8 from willyw0nka/fix-builder

Fixed builder

Commit:
Added pareto principle achievement

rebase to development

Commit:
Added pareto principle achievement

willyw0nka commented:
Thank you for your contribution.
The pull request needs to target development branch, take a look at CONTRIBUTING.md that I just created.
You will need to format the docstrings too, check any docstring on the develoment branch.

Commit:
better docstrings

Ciock committed:
Merge branch 'development' into pareto-principle-achievement

Ciock commented:
Here I am,
I hope I haven't bothered by rebasing this branch on development, but I thought that the merge would be easier since all the docstrings have changed.

Anyway I changed the target branch of the pull request and updated the docstrings.

I apologize if I didn't wait to discuss the issue first, however don't feel obliged to insert the change on the main branch, especially not before there is another achievement to align the output image.
